generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts

```diff
@@ -70,4 +80,8 @@ import {
         <%=jhiPrefixCapitalized%>AlertErrorComponent
     ]
 })
-export class <%=angularXAppName%>SharedCommonModule {}
+export class <%=angularXAppName%>SharedCommonModule {
+    constructor() {
+        registerLocaleData(locale);
```

Comment:
Just ensure that this works fine with AOT as it has history of not working with normal js constructs sometimes

Reply:
hum :), I don't know what this really means... can you guide me a bit on this ? thx

Reply:
just run the app using `./mvnw -Pprod` and make sure everything works fine, as this will use AOT and if there is an issue here it should fail

Reply:
So everything seems to work fine with this. Running `./mvnw -Pprod`compiles and runs with no problem. Then in the app everything is fine as well, I tried with different locales, so it's all good for me.